### ISSUE TITLE
'urinary system disease' and 'disease of genitourinary system'

### ISSUE THREAD

**Issue by paolaroncaglia** [EBISPOT/efo#641]
Stemming from https://github.com/EBISPOT/efo/issues/492#issuecomment-557550818
(making this into a new ticket for convenience):

Currently, EFO has 'urinary system disease' and 'disease of genitourinary system' as siblings.
For the EFO profile for Open Targets Therapeutic Areas, we were required to remove the latter and keep the former as a TA.
To make things consistent between EFO3 and EFO OT TA profile, the initial suggestion was to make 'urinary system disease' a subclass of 'disease of genitourinary system'.
However, the underlying Uberon structure opposes this. I raised the issue with Uberon (see obophenotype/uberon#1533). They replied with the following proposal:
"The genital and urinary systems should be separate and distinct. We could add some in_taxon statements to clarify their interconnected anatomy in mammals and other taxonomic groups, and specify differences in males and females." (obophenotype/uberon#1533 (comment))
That might fix issues for Mondo and EFO, we'll need to wait and see.

**Comment by paolaroncaglia** [comment]
@zoependlington the underlying Uberon issue hasn't been resolved yet (though there's been some discussion). So I guess that you'd want to keep this EFO ticket open.

**Comment by jamesamcl** [comment]
**Status triage (2026-08-26).** This ticket was waiting on the Uberon genital-vs-urinary discussion (obophenotype/uberon#1533). Six years on, the landscape has changed enough that the original question appears resolved:

- **'urinary system disease' (EFO:0009690) has been obsoleted** (EFO 3.88.0) and replaced by **MONDO:0002118 'urinary system disorder'**, which is active in EFO. So the sibling pair described here no longer exists.
- **MONDO has settled the underlying question**: MONDO:0021145 'disorder of genitourinary system' (the xref of EFO:0009663) is now **obsolete in MONDO**, with the rationale "out of scope – grouping term. Terms to consider: 'reproductive system disorder' (MONDO:0005039), 'urinary system disorder' (MONDO:0002118)". That matches Uberon's 2019 proposal to keep the genital and urinary systems separate and distinct (uberon#1533 itself is still open, but its outcome no longer blocks EFO).
- **EFO:0009663 'disease of genitourinary system' is still active in EFO**, but it now has **no subclasses** in the released ontology, its MONDO counterpart is obsolete, and nothing in efo-edit.owl depends on it. (In the current release it is also classified under 'urogenital neoplasm' (EFO:0003863), which looks like a release artifact and is a further sign the term is no longer doing useful work.)

**Recommendation:** treat the original request as superseded by the MONDO/Uberon resolution, obsolete EFO:0009663 in line with MONDO:0021145 (pointing users to MONDO:0002118 and MONDO:0005039 as the replacements-to-consider), and close this ticket. If Open Targets still needs a combined genitourinary therapeutic-area grouping, that would now be a profile-level concern rather than an EFO hierarchy one.

---
*This is an automated triage review by an AI agent; maintainers, please correct if anything here is wrong.*
